Sources/SpeziHealthKit/CollectSample/CollectSample.swift

```diff
@@ -11,6 +11,30 @@ import Spezi
 
 
 /// Collects a specified `HKSampleType`  in the ``HealthKit`` module.
+///
+/// This structure define what and how the HealthKit samples are collected. By default, all samples of the provided `HKSampleType` after you provide the application authorization to collect the samples will be collected when the ``HealthKit/triggerDataSourceCollection()`` function is called.
```

Comment:
I think this sentence is slightly off, it might make sense to divide it up into two? In addition the collection starts on calling `triggerDataSourceCollection` if you configure it as manual or automatic once the application is launched with you use anything else than manual.